test.py

```diff
@@ -675,7 +675,10 @@ async def run(self, options: argparse.Namespace) -> Test:
 
         def set_summary(summary):
             self.summary = summary
-            logging.info("Test %s %s", self.uname, summary)
+            if self.success:
+                logging.info("Test %s %s", self.uname, summary)
```

Comment:
A style comment, it could be more brief with something like log = logging.info if self.success else logging.error

Reply:
Changed in https://github.com/scylladb/scylladb/commit/9b65448d38bd39a94173902c0afa117362fd44d4